Readability.js

```diff
@@ -961,6 +961,18 @@ Readability.prototype = {
           return null;
         }
       } else {
+        // Find out text direction from ancestors of final top candidate.
+        var ancestors = [parentOfTopCandidate].concat(this._getNodeAncestors(parentOfTopCandidate));
```

Comment:
I would say we don't need to do that.

If `topCandidate` has `dir` attribute, we will just see the result generated by Reader Mode algorithm with correct text direction. For example, the result could be
```html
<div id="readability-page-1" class="page">
    <div dir="rtl"> <!-- The Top  Candidate-->
```

And also notice that at that time, all `topCandidate` nodes are already children of `articleContent` node[1]. If we want to check the `dir` of `topCandidate`, it means we check `articleContent` element for getting text direction. We might not need to do that.

What do you think?

[1]: https://github.com/mozilla/readability/blob/master/Readability.js#L906